snuba/datasets/errors_replacer.py

```diff
@@ -241,6 +244,9 @@ def process_message(self, message: ReplacementMessage) -> Optional[Replacement]:
         type_ = message.action_type
         event = message.data
 
+        if type_ in REPLACEMENT_EVENT_TYPES:
+            metrics.increment(type_, 1)
```

Comment:
I would advise you put the type in a tag so we can easily get the metric grouped by tag and use the metric name to identify the action (like "execute"). This will make your metric consistent with what we generally have in dashboards. The name identifies an action or what the metric stands for and a tag allows you to break down the metric itself.

```
metrics.increment("process", tags= {"type": type_})
```